apps/server/src/auth/auth.service.ts

```diff
@@ -1,28 +1,98 @@
 import { Request } from 'express';
 
-import { Injectable } from '@nestjs/common';
+import { Repository } from 'typeorm';
+
+import { JwtService } from '@nestjs/jwt';
+import { InjectRepository } from '@nestjs/typeorm';
+import { HttpStatus, Injectable } from '@nestjs/common';
+
+import { RoleEnum } from '@/common/constants/role.enum';
+import { ErrorMessage } from '@common/errors/error.message';
+import { RestException } from '@common/exceptions/rest.exception';
+import { ErrorDescription } from '@common/errors/constants/description.error';
+
+import { User } from '@/user/entities/user.entity';
+
+import { generateAccessToken } from '@/auth/util/jwt.util';
+import { UserDto } from '@/user/dto/user.dto';
+
 @Injectable()
 export class AuthService {
+  constructor(
+    @InjectRepository(User) private userRepository: Repository<User>,
+    private readonly jwtService: JwtService,
+  ) {}
```

Comment:
This is anti-pattern.
Let's refactor this to use `UserService` instead of `UserRepository`.

Reply:
updated to use `UserService`